Makefile

```diff
@@ -24,6 +24,9 @@ lint-yaml: normalize-yaml
 normalize-yaml: # Normalizes YAML files (alphabetizes keys, fixes line length, smart quotes)
 	npm run normalize-yaml
 
+normalize_yaml: # Normalizes YAML files (alphabetizes keys, fixes line length, smart quotes)
+	npm run normalize-yaml
+
```

Comment:
Was this intended to be included? Could we make it a proper "alias" by having it just call the other Make target?

```suggestion
normalize_yaml: normalize-yaml
```

Reply:
oops, no